specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -482,79 +488,79 @@
       "type": "object",
       "properties": {
         "memoryInGB": {
-          "description": "The memory limit in GB of this container.",
+          "description": "The memory limit in GB of this container instance.",
           "type": "number",
           "format": "double"
         },
         "cpu": {
-          "description": "The CPU limit of this container.",
+          "description": "The CPU limit of this container instance.",
           "type": "number",
           "format": "double"
         }
       }
     },
     "AzureFileVolume": {
-      "description": "The Azure file volume.",
+      "description": "The properties of the Azure File volume.",
       "type": "object",
       "required": [
         "shareName",
         "storageAccountName"
       ],
       "properties": {
         "shareName": {
-          "description": "The Azure file share name.",
+          "description": "The name of the Azure File volume.",
           "type": "string"
         },
         "readOnly": {
-          "description": "The flag indicating whether the Azure file volume is read only.",
+          "description": "The flag indicating whether the Azure File volume is read-only.",
           "type": "boolean"
         },
         "storageAccountName": {
-          "description": "The storage account name to access to the Azure file.",
+          "description": "The name of the storage account that contains the Azure File volume.",
```

Comment:
Another volume vs. share comment, Azure Files does not have volumes, but shares. We definitely want to be careful not to mix terms. I'll refrain from further volume/share comments, you get the idea. :)